i18n/fr/docusaurus-plugin-content-docs/version-25.10/upgrade/upgrade-mysql.md

```diff
@@ -0,0 +1,160 @@
+---
+id: upgrade-mysql
+title: Mettre à jour MySQL
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Si vous utilisiez Centreon 24.04 ou 24.10 avec MySQL, sachez que le support de MySQL 8.0 prendra fin en avril 2026. Assurez-vous que votre Centreon utilise toujours [une version supportée de MySQL](#versions-de-mysql-compatibles-avec-centreon-2510).
+
+Vous pouvez potentiellement avoir plusieurs bases MySQL dans votre architecture. Le serveur central a une base, chaque serveur distant a une base, et les modules MBI et MAP ont chacun une base dédiée. Dans une architecture, toutes les bases doivent avoir la même version de MySQL.
+
+## Processus de mise à jour
+
+Ne mettez pas à jour votre version de Centreon et votre version de MySQL en même temps. Vous pouvez effectuer les deux mises à jour l'une après l'autre dans l'ordre que vous souhaitez, mais assurez-vous que la première procédure de mise à jour soit entièrement terminée avant d'effectuer la seconde.
+
+<!--If you are upgrading from one major version of Centreon to another at the same time as upgrading MySQL (e.g. upgrading from Centreon 24.04 to Centreon 25.10), you must:
+
+1. Upgrade Centreon (packages, web installation, deploying the configuration).
+2. Upgrade MySQL.
+
+> Refer to the official MySQL documentation for more information about this process:
+> http://dev.mysql.com/doc/refman/8.4/en/upgrade-binary-package.html-->
```

Comment:
This is strangely displayed in the github preview, you might want to check 
<img width="1165" height="193" alt="image" src="https://github.com/user-attachments/assets/1da7f844-4230-455d-856e-f9c60c376c32" />
(same for english version)